Gemfile

```diff
@@ -3,19 +3,18 @@ source "http://rubygems.org"
 # try to slowly migrate to v2 of the aws api
 gem 'aws-sdk', '~> 2'
 
-gem 'dynect_rest', '= 0.4.6'
```

Comment:
We no longer seem to do anything with this gem.

Reply:
build failure suggests we need to remove a reference to it